com.unity.render-pipelines.universal/Runtime/Passes/MainLightShadowCasterPass.cs

```diff
@@ -169,11 +169,14 @@ void RenderMainLightCascadeShadowmap(ref ScriptableRenderContext context, ref Cu
 
                 for (int cascadeIndex = 0; cascadeIndex < m_ShadowCasterCascadesCount; ++cascadeIndex)
                 {
+                    //settings.splitData = m_CascadeSlices[cascadeIndex].splitData; // NOTE: currently DrawShadows culls more casters if no ShadowSplitData.cullingPlanes are set (version cds 8652678b), so it is currently better to not pass the m_CascadeSlices[cascadeIndex].splitData object returned by CullingResults.ComputeDirectionalShadowMatricesAndCullingPrimitives (change introduced in 8bf71cf). Culling is only based on the ShadowSplitData.cullingSphere distances.
```

Comment:
I would like to understand this a bit better. Are you sure the additionally culled casters are valid? And no artifacts are produced.

Reply:
This comment summarizes the conclusion I came to after shortly experimenting with [ScriptableRenderContext.DrawShadows](https://docs.unity3d.com/2020.2/Documentation/ScriptReference/Rendering.ScriptableRenderContext.DrawShadows.html) parameters.
If you would like to investigate further this experiment, I uploaded the test projects I used, and added a bit of explanations, here:
https://github.com/Wilfrid-Unity/main-light-shadow-cascades-culling

Reply:
Should this be filed as a bug?

Reply:
I think so. I filed this issue [1273362](https://issuetracker.unity3d.com/product/unity/issues/guid/1273362) about it.